xcode/Subconscious/Shared/Components/Common/Transclude/TranscludeView.swift

```diff
@@ -2,41 +2,95 @@
 //  TranscludeView.swift
 //  Subconscious
 //
-//  Created by Gordon Brander on 8/23/22.
+//  Created by Gordon Brander on 2/22/23.
 //
 
 import SwiftUI
 
+struct ExcerptView: View {
+    var excerpt: String
+    var lineLimit: Int = 5
+    var spacing: CGFloat = AppTheme.unit
+    var excerptLines: [EnumeratedSequence<[String.SubSequence]>.Element] {
+        Array(excerpt.split(separator: "\n").enumerated())
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: spacing) {
+            ForEach(excerptLines, id: \.offset) { idx, line in
+                Text("\(String(line))")
+                    .fontWeight(idx == 0 && excerptLines.count > 1 ? .medium : .regular)
```

Comment:
Only one line? Regular weight, full opacity.

Two lines: the first is medium, full opacity, the second is regular weight and 0.8 opacity. On light mode the weight is enough but in dark mode we need more constrast.

Reply:
Feels like we should find a treatment that can work in light and dark mode. Medium weight seems fine to me for first block.

Reply:
Removed the opacity tweak.